packages/expo/src/react-meta-tags.native.tsx

```diff
@@ -0,0 +1,70 @@
+// Shim for supporting React 19 meta tags on native.
+import * as ReactNativeViewConfigRegistry from 'react-native/Libraries/Renderer/shims/ReactNativeViewConfigRegistry';
+
+for (const key of ['title', 'link', 'meta', 'style', 'script']) {
+  ReactNativeViewConfigRegistry.register(key, () => {
+    const config = ReactNativeViewConfigRegistry.get('RCTVirtualText');
+    return {
+      ...config,
+      // Modify to make nothing render
+      validAttributes: {
+        ...config.validAttributes,
+        // Add any custom attributes you want to support
+        // For example, if you want to support a custom attribute called "title"
+        children: {
+          process: () => {
+            return null;
+          },
+        },
+      },
+    };
+  });
+}
```

Comment:
i'm not convinced about supporting these tags, imo if we error on some of the dom element tags we should likely behave the same for all.